router/batchrouter/asyncdestinationmanager/snowpipestreaming/types.go

```diff
@@ -149,3 +163,35 @@ func (e *event) setUUIDTimestamp(formattedTimestamp string) {
 		e.Message.Data[uuidTimestampColumn] = formattedTimestamp
 	}
 }
+
+func newAuthzBackoff(initialInterval time.Duration, clock backoff.Clock) *authzBackoff {
+	return &authzBackoff{
+		options: []backoff.ExponentialBackOffOpts{
+			backoff.WithInitialInterval(initialInterval),
+			backoff.WithMultiplier(2),
+			backoff.WithClockProvider(clock),
+			backoff.WithRandomizationFactor(0),
+			backoff.WithMaxElapsedTime(0),
+			backoff.WithMaxInterval(time.Hour),
```

Comment:
Can we make these configurable instead of hard coding it?

Reply:
I've made them configurable except for randomizationFactor and maxElapsedTimeas, as I felt there is no need to configure them. Let me know if you feel otherwise